src/scicookie/{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -40,6 +40,16 @@ repos:
       types:
         - python
 {% endif %}
+{%- if cookiecutter.use_ruff == 'ruff' %}
```

Comment:
@ayeankit could you change the order of the parameters here, let's keep all options with the same structure.

And probably you can remove file.

As a follow up I will apply the same structure to the pre commit config in rhe root of the project